spring-cloud-gcp-autoconfigure/pom.xml

```diff
@@ -126,5 +119,11 @@
             <optional>true</optional>
         </dependency>
 
+        <!-- Config -->
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-context</artifactId>
```

Comment:
I set this dependency to optional=false, so users don't have to.
Rationale is that I see `spring-cloud-context` as something very broad, and the link to Spring Cloud GCP Config is not straightforward, but I might be wrong.
Thoughts?

Reply:
Makes sense to me as long as nothing else (except config) depends on it. But I think that might not quite be true (sleuth and stream depend on it as well). That's why I think we need the `@EnableGcpConfiguration` annotation. But we can continue that discussion elsewhere.

Reply:
We've already got `@ConditionalOnProperty` for Config, that should cover it for now?
We never had any `@ConditionalOnClass` for a `spring-cloud-context` class, anyway.